zstd-safe/zstd-sys/src/wasm_shim.rs

```diff
@@ -14,17 +29,33 @@ pub extern "C" fn rust_zstd_wasm_shim_calloc(
     nmemb: usize,
     size: usize,
 ) -> *mut c_void {
+    // the strategy for rust_zstd_wasm_shim_malloc is used, described there
```

Comment:
Should we maybe just call `rust_zstd_wasm_shim_malloc(nmemb * size)`?

Reply:
Sounds good, although this prompted me to check the c docs: turns out calloc expects its memory to be zeroed.